src/content/basics/nodepools/index.md

```diff
@@ -113,6 +113,35 @@ Pay close attention to the workloads being rescheduled on other nodes once nodes
 
 See the [`gsctl delete nodepool`](/reference/gsctl/delete-nodepool/) reference for how to delete a node pool using the CLI.
 
+## Instance distribution
+
+Node pools can contain a mix of [on-demand](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-on-demand-instances.html) and [spot instances](https://aws.amazon.com/ec2/spot/) that will allow you to optimize your cost. 
+
+There are two parameters that will allow you to configure which instances are going to be used:
+
+- *On-demand base capacity*: controls how much of the initial capacity is made up of on-demand instances.
+
+- *Spot instance percentage above base capacity*: controls the percentage of spot instances to be used for worker nodes beyond the number of *on-demand base capacity*.
+
+### Configuration examples
+
+| Configuration                                        | Total Instances  | On-Demand | Spot 
+|------------------------------------------------------|------------------|-----------|------
+| On-demand base capacity: 0<br>Spot instance percentage: 0     | 20               | 20        | 0    
+| On-demand base capacity: 10<br>Spot instance percentage: 50   | 20               | 15        | 5    
+| On-demand base capacity: 5<br>Spot instance percentage: 100   | 20               | 5         | 15   
+| On-demand base capacity: 0<br>Spot instance percentage: 100   | 20               | 0         | 20   
+
+## Using similar instance types
+
+Using multiple instance types in a node pool has some benefits for on-demand and spot instances:
+- Using multiple instance type allows better price optimization. Popular instance types tend to have more price adjustments. Picking older-generation instance types that are less popular tends to result in lower costs and fewer interruptions.
+- AWS has a limited number of instances per Availavility Zone and could be that your selected instance are temporarily out of stock, allowing the node pool to use multiple instance types would decrease the probability of not being able to scale up.
+
+Instances that contain the same amount of CPU and RAM are considered similar, for example if you select `m5.xlarge` then the node pool can fall back on `m4.xlarge` too if needed.
+
+You can check the instance types that are considered to be similar in the [aws-operator](https://github.com/giantswarm/aws-operator/blob/master/service/controller/key/machine_deployment.go#L15).
```

Comment:
For me, that link doesn't seem to point to the right place. Maybe use a URL to a commit version instead?

Reply:
It's poiting to where it will end up, as the operator has not been merged to master yet